composer.json

```diff
@@ -175,10 +176,9 @@
     },
     "require-dev": {
         "drupal/coder": "^8.2",
-        "drupal/core-dev": "9.5.9",
+        "drupal/core-dev": "^10",
```

Comment:
This can be removed now (or create a followup story)

Reply:
Good call! I've removed "core-dev"